voice/rateLimits.md

```diff
@@ -1,5 +1,5 @@
 # Rate Limits
 
-Rate limits for the Voice API are set by account by your customer account manager.  By default, rate limits are set to 3 CPS and 30 active sessions.
+Rate limits for the Voice API are set by account by your customer account manager. By default, rate limits are set to 3 CPS and 30 active sessions. If you think you may need additional voice capacity, please open a ticket with our Account Management team. 
```

Comment:
`Account Management team` seems weird to me, but :man_shrugging:
Would `Customer Representative` or similar be appropriate, customer-facing verbiage?

Reply:
I was given this verbiage from them.